frontend/src/routes/dashboard/+page.svelte

```diff
@@ -41,18 +49,41 @@
 		});
 	}
 
-	function addTask(taskDescription, assigneeName) {
-		tasks.update((currentTasks) => {
-			const newTask = {
-				id: currentTasks.length + 1,
-				description: taskDescription,
-				assignedTo: assigneeName,
-				completed: false
-			};
-			return [...currentTasks, newTask];
-		});
-		taskInput = '';
-		assignedInput = '';
+	async function addTask(taskDescription, assigneeName) {
+		const currentGroup = $groupId;
```

Comment:
One minor issue is that, IIRC, reactive `$store` syntax doesn't work inside functions. You can use `get(store)` instead to get the value at the time of the call.